nle/dataset/dataset.py

```diff
@@ -128,7 +128,7 @@ def _ttyrec_generator(
         key_vals = [
             ("tty_chars", chars),
             ("tty_colors", colors),
-            ("tty_cursors", cursors),
+            ("tty_cursor", cursors),
```

Comment:
why singular if the variable name is plural?

Reply:
singular because we chose the same keys as NLE. There we have 1 frame of 80 x 24 chars, 1 frame of 80 x 24 colors, and 1 cursor (2 coords). In this case we technically have a batch of these. For consistency, it might be better to move to `timestamp` and `gameid` for the other keys.